docs/Dialects/onnx.md

```diff
@@ -4253,6 +4240,27 @@ ONNX Resize operation
 | :----: | ----------- |
 `Y` | tensor of 8-bit unsigned integer values or tensor of 16-bit unsigned integer values or tensor of 32-bit unsigned integer values or tensor of 64-bit unsigned integer values or tensor of 8-bit signless integer values or tensor of 16-bit signless integer values or tensor of 32-bit signless integer values or tensor of 64-bit signless integer values or tensor of 16-bit float values or tensor of 32-bit float values or tensor of 64-bit float values or tensor of stirng type values or tensor of 1-bit signless integer values or tensor of complex type with 32-bit float elements values or tensor of complex type with 64-bit float elements values or memref of any type values
 
+### `onnx.Return` (::mlir::ONNXReturnOp)
```

Comment:
quick question: these changes come from upgrading onnx to a new version?

Reply:
Standard return op specifies that its parent must be the mlir::FuncOp, so we need a different terminator op for subgraph (imported as region now).
https://github.com/llvm/llvm-project/blob/dfb558b82d3f2ca228799b03d2be00b163d04ea5/mlir/include/mlir/Dialect/StandardOps/IR/Ops.td#L2295